container_pipeline/workers/base.py

```diff
@@ -29,16 +29,17 @@ def notify(self, data):
         """
         self.queue.put(json.dumps(data), 'master_tube')
 
-    def export_logs(self, logs, destination):
+    def export_logs(self, logs, destination, json_logs=False):
```

Comment:
How we are going to read the file? if while reading the file we are expecting it to be JSON, lets write it to JSON format only. IMO, I dont see the value for putting the condition while writing it, and not considering while reading.